locales/en/connection-tab.json

```diff
@@ -0,0 +1,29 @@
+{
+  "admin_url_description": "Connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access. ",
+  "admin_url_label": "URL for Kafka Admin REST API",
+  "admin_url_popover_body": "Use the REST API URL to connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access.",
+  "admin_url_popover_label": "URL for Kafka Admin REST API",
+  "admin_url_button_aria_label": "More information about the URL for Kafka Admin REST API",
+  "authentication_method": "Authentication method",
+  "bootstrap_server": "Bootstrap server",
+  "bootstrap_server_description": "Establish the initial connection between your client and Kafka instance.",
+  "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the <value>Service Accounts</value> page.",
+  "current_instance": "Accounts do not have access to this instance unless granted under the instance’s <value>Access tab</value>. The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+  "drawer_resource_tab_body_description_1": "Your application or tool initially connects to the Kafka instance using the bootstrap server and authenticates with your service account credentials. You can also access the Kafka instance by calling the REST API using a token obtained from Red Hat SSO.",
+  "kafka_service_fleet_manager_button_aria_label": "More information about the URL for Kafka Service Fleet Manager",
+  "kafka_service_fleet_manager_description": "Connect to the control plane to manage Kafka instances and service accounts, and to access metrics.",
```

Comment:
@wtrocki Should  also need to replace the term 'control plane' used here

Reply:
My personal opinion: yes
Control plane is some kubernetes pattern that is not known outside kubernetes.

@jgiardino voice matters in terms of naming.

Reply:
I agree that _control plane_ is a term we should avoid.

We can maybe solve this problem by eliminating the "Connect to..." phrase entirely:

- "Configure topics, consumer groups, and access for an individual Kafka instance."
- "Manage Kafka instances and service accounts, and access metrics."

WDYT?

Reply:
Also, I don't see where the short description for the **REST APIs** section is defined, but we probably need to update that, too. Maybe something like the following:

"Streams for Apache Kafka provides two REST APIs, one for configuring individual Kafka instances, and one for managing resources such as Kafka instances and services accounts."

Reply:
@jbyrne-redhat, attaching the screenshot for REST APIs short description
![image](https://user-images.githubusercontent.com/53568062/175075494-d5b3d66a-26bd-4b0b-9a3e-0c16bc1601a0.png)

Reply:
Thanks, @HemaHG  - yeah I saw the short description in the Storybook preview that Jenn pointed me to. I saw that we're using the term _control plane_ there too, so suggested some alternative wording ^^.

Reply:
Thanks, @jbyrne-redhat for reviewing. I have updated the PR accordingly https://619e1921b3e38b003afeb2be-csichqiapa.chromatic.com/?path=/story/kafka-kafkainstancedrawer-kafkaconnectiontab--connection-tab